docs/advanced/checked-build.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Checked Builds
+redirect_from:
+  - /Checked_Build/
```

Comment:
The page is always at http://www.mono-project.com/docs/advanced/checked-build/, if you add the redirect_from it'll just redirect there from http://www.mono-project.com/Checked_Build/